samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/SectionContentsBean.java

```diff
@@ -520,7 +411,7 @@ public void setMetadataRandowDraw(SectionDataIfc section) {
 		String poolname = section.getSectionMetaDataByLabel(SectionDataIfc.POOLNAME_FOR_RANDOM_DRAW);
 		if (SectionDataIfc.RANDOM_DRAW_FROM_QUESTIONPOOLS.equals(Integer.valueOf(section.getSectionMetaDataByLabel(SectionDataIfc.AUTHOR_TYPE))) && 
 				section.getSectionMetaDataByLabel(SectionDataIfc.RANDOM_POOL_COUNT) != null) {
-			Integer count = Integer.valueOf(section.getSectionMetaDataByLabel(SectionDataIfc.RANDOM_POOL_COUNT));
```

Comment:
There are more Integer.valueOf on this file... e.x
https://github.com/sakaiproject/sakai/pull/13395/files#diff-427b06a8f86a14d72b1c42d4ee13021b6e85bcff920cc427ee67bec0548e6a12R400

Reply:
ok all the available ones are done.